admission-webhook/deploy/create-signed-cert.sh

```diff
@@ -109,7 +109,7 @@ fi
 
 # create server cert/key CSR and send to k8s API
 CSR_CONTENTS=$(cat <<EOF
-apiVersion: certificates.k8s.io/v1beta1
+apiVersion: certificates.k8s.io/v1
```

Comment:
@vitaliy-leschenko @jsturtevant 

I believe that this change will cause issues, since `v1beta1` is not compatible with `v1`.

Looking at the [official docs](https://kubernetes.io/docs/reference/access-authn-authz/certificate-signing-requests/#request-signing-process), I found:
```
Note that spec.signerName is a required key after API version certificates.k8s.io/v1.
```

Also, I tried deploying the GMSA webhook, and the CSR resource fails with:
```
error validating data: ValidationError(CertificateSigningRequest.spec): missing required field "signerName" in io.k8s.api.certificates.v1.CertificateSigningRequestSpec; if you choose to ignore these errors, turn validation off with --validate=false
```

Reply:
yes, I ran into that: https://github.com/kubernetes-sigs/windows-gmsa/pull/31